### PR TITLE
Corrections and efficiency updates

### PR DESCRIPTION
**Pull request by Spydar007**

### CODE REVIEW

File: ipsw.sh [hunk 1]
```diff
@@ -68,52 +68,52 @@ LLB_patch=$(buddy LLB:Patch)
 iBoot_patch=$(buddy iBoot:Patch)
 DeviceTree_patch=$(budd DeviceTree:Patch)
```
Comment (TheonlyGermanGuy) [inline ipsw.sh]: "budd" needs to be changed to "buddy"

File: ipsw.sh [hunk 2]
```diff
@@ -68,52 +68,48 @@ LLB_patch=$(buddy LLB:Patch)
...
+bspatch "${LLB}.dec" "${FirmwarePath}LLB.img3" "${LLB_patch}"
+bspatch "${iBoot}.dec" "${FirmwarePath}iBoot.img3" "${iBoot_patch}"
+bspatch "${DeviceTree}.dec" "${FirmwarePath}DeviceTree.img3" "${DeviceTree_patch}"
 
 echo "Editing manifest"
 
-echo "LLB.img3" >> ${manifest}
-echo "iBoot.img3" >> ${manifest}
-echo "DeviceTree.img3" >> ${manifest}
+echo "LLB.img3" "iBoot.img3" "DeviceTree.img3" >> "${manifest}"
```
Comment (TheonlyGermanGuy) [inline ipsw.sh]: Using this method causes errors in the manifest. (Words are written in one line, not in multiple)

File: ipsw.sh [hunk 3]
```diff
@@ -68,52 +68,48 @@ LLB_patch=$(buddy LLB:Patch)
...
 
 echo "Cleaning up"
 
-rm -f ${FirmwarePath}*.dec
+rm -f "${FirmwarePath}*.dec"
```
Comment (TheonlyGermanGuy) [inline ipsw.sh]: Using quotes doesn't seem to work with "rm". Using just the variable works as wanted.

**Comment by TheonlyGermanGuy** [comment]
There are some errors in your branch. See my comments for more information

File: ipsw.sh (hunk 1 above)
Comment (Spydar007) [inline ipsw.sh]: This is your mistake, not mine, but I have corrected it anyway :smile:

File: ipsw.sh (hunk 3 above)
Comment (Spydar007) [inline ipsw.sh]: There is no reason why this shouldn't work.

**Comment by Spydar007** [comment]
New commits have been added to fix these errors.

File: ipsw.sh (hunk 3 above)
Comment (TheonlyGermanGuy) [inline ipsw.sh]: I don't know why it doesn't work. At least on my machine it doesn't.

Comment (Spydar007) [inline ipsw.sh]: Removed them for now, will investigate further.

**Comment by Spydar007** [comment]
Closing for now, script doesn't copy to ipsw. Will refork and correct when I have time.

**Pull request closed without merging**